### DISCUSSION TITLE
Home position not useful enough )

### DISCUSSION THREAD

**Discussion by bimawa** [1coderookie/Klipper4Kobra2series#5] [Ideas]
Home position for my Kobra2neo is in the center of the table. Each start the print program nozzle, leave a drop before wiping it. So each time I should remove this. So I found a better solution. 
Check it, please. Maybe I may make a PR?
```
[safe_z_home]
home_xy_position: 10, 20
speed: 50.0
z_hop: 10.0
z_hop_speed: 15.0
move_to_previous: False
```
I changed the save_z_home, and now it moved nozzle to the left-bottom before start and makes my the table clean for future models

**Comment by 1coderookie** [comment]
Thanks - which cfg exactly are you referring to?  
Actually the most 'useful' home position is 0,0 actually - I'll change it accordingly then.

**Reply by bimawa** [reply]
printer.cfg

**Reply by 1coderookie** [reply]
Yeah sure - but which one? I have several subdirectories for different printer models.. ;)  
But I just found it.  
The thing is that it's not a cfg file I made, and therefore I'll leave it like it is.  
If *I* personally would use this, then I'd change the safe home position to 0,0 anyways. So just set it to whichever position you personally prefer..   
Thx for reahcing out tho.

**Reply by bimawa** [reply]
Oh yea sorry. Kobra2neo
Let's make labels with models ^_^

**Reply by bimawa** [reply]
> If I personally would use this, then I'd change the safe home position to 0,0 anyways. So just set it to whichever position you personally prefer..

Yes Discusses created for it. It's why I not make  a PR. I not sure about it

**Reply by 1coderookie** [reply]
Yes, thanks for that ;)  

Well, the reason I personally alway use 0,0 as the home position is because I like to have the bed out of the way when a print starts.  
I use to babysit the machine when it starts a print and pull away the leaking filament with some long tweezers.  
Since I mostly print PETG with 80°C bedtemp and heat up the bed and let it settle for ~15min before actually starting a print, I don't burn myself with the bed being all the way to the back and the head being all the way to the left when doing so.. ;)
